addons/binding/org.openhab.binding.tellstick/README.md

```diff
@@ -29,7 +29,7 @@ The attentive reader discovers that there is many missing sensor types; `UV`, `L
 Support have not been implemented on the openhab side yet, contributions are welcome.  
 
 ***Switchbased sensors workaround*** <br>
-*   Some 433MHz magnetic & PIR sensors for example magnetic door sensors are detected as a regular `switch` things instead of a separate type. There is technically no way of distinguish them apart from regulur `switch` things.
+*   Some 433MHz magnetic & PIR sensors for example magnetic door sensors are detected as a regular `switch` things instead of `contact` type. There is technically no way of distinguish them apart from regulur `switch` things.
```

Comment:
Consider, "Some 433MHz magnetic & PIR sensors, for example, magnetic door sensors, are detected as regular `switch` Things instead of type `contact`."  "There is no way ofdistinguishing them from regular `switch` things."